backend/src/kitconcept/intranet/upgrades/configure.zcml

```diff
@@ -73,5 +73,6 @@
   <include package=".v20250808001" />
   <include package=".v20250818001" />
   <include package=".v20250825001" />
+  <include package=".v20250828001" />
```

Comment:
@iFlameing There's no reason to add an entire new folder for an upgrade step that doesn't have any custom Python code. Just put the genericsetup:upgradeSteps directive right here, like the ones above.

Reply:
@davisagli @ericof told me to always create a new folder.